IsIt5Yet.html

```diff
@@ -13,6 +13,15 @@
   </head>
   <body>
     <h1 class="sr-only">Is it 5 yet?</h1>
-    <p class="text-center transform translate-y-96 text-6xl text-white"></p>
+    <p
+      id="isIt5Yet"
+      class="text-center transform translate-y-96 text-6xl text-white"
+    ></p>
+    <div class="absolute -bottom-10 text-4xl text-red-600">
+      <a
+        href="https://www.aa.org/pages/en_us/information-on-alcoholics-anonymous#:~:text=Alcoholics%20Anonymous%20is%20an%20international,no%20age%20or%20education%20requirements."
+        >Get You Some Help!</a
```

Comment:
The user can't see where the link goes to. Change this to be more like "here are some helpful resources you fuckin drunk." or "We understand you might have a problem drinking .....  here are some resources".